### ISSUE TITLE
Error setting up a collection with generative.anthropic()

### ISSUE THREAD

**Issue by ffleandro** [weaviate/typescript-client#197]
I'm using the typescript `weaviate-client` v3.1.4 to create a RAG collection with anthropic.

Using weaviate v1.26.4 and confirmed the weaviate meta shows the `generative-anthropic` module enabled.

This code throws `TypeError: generative.anthropic is not a function`:
```
import weaviate, { vectorizer, generative, configure } from 'weaviate-client';
await client.collections.create({
   ... 
   generative: generative.anthropic(),
   ...
});
```

Confirmed using `generative.openAI()` works fine.

**Comment by tsmith023** [comment]
Hi @ffleandro, I closed the issue as completed since the code is in `main` but I've not yet released `v3.1.5`. I will do so this morning, however, once one last issue is fixed 😁

**Comment by tsmith023** [comment]
Scratch that actually, I have released `v3.1.5` that includes this fix. Cheers!
